scripts/get_container_images.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
```

Comment:
set -e ?

Reply:
Not really useful since I already check all the return codes of the executed commands, but I can add a "set -eu" anyway.